hal/src/main/native/athena/HAL.cpp

```diff
@@ -347,7 +347,7 @@ HAL_Bool HAL_Initialize(int32_t timeout, int32_t mode) {
 
   FRC_NetworkCommunication_Reserve(nullptr);
 
-  std::atexit([]() {
+  std::atexit([](void) {
```

Comment:
This one should not be changed.

Reply:
I'm not sure how that slipped in. The latest version of the wpiformat PR doesn't do that. I'll modify it to replace (void) with () for non-C cases as well.